test/e2e/node_reuse_test.go

```diff
@@ -517,10 +516,9 @@ func untaintNodes(clientSet *kubernetes.Clientset, nodes *corev1.NodeList, taint
 	count = 0
 	for i := range nodes.Items {
 		Logf("Untainting node %v ...", nodes.Items[i].Name)
-		newNode, changed, err := taints.RemoveTaint(&nodes.Items[i], taint)
-		Expect(err).To(BeNil(), "Failed to remove taint")
+		newNode, changed := removeTaint(&nodes.Items[i], taint)
 		if changed {
-			node, err := clientSet.CoreV1().Nodes().Update(ctx, newNode, metav1.UpdateOptions{})
+			_, err := clientSet.CoreV1().Nodes().Update(ctx, newNode, metav1.UpdateOptions{})
```

Comment:
Why do we need to remove the node variable? It may fail at line 525

Reply:
I screwed up in the rebase perhaps. Let me take a look

Reply:
good catch fixed @namnx228